replication.go

```diff
@@ -345,10 +369,42 @@ func (c *Client) SyncPolicy(ctx context.Context, policyName string) error {
 			Id: policyName,
 		}
 		log.Info(ctx, "found no active sync jobs, starting a new one")
-		_, err := c.StartSyncIQJob(ctx, jobReq)
-		if err != nil {
-			return err
+
+		// workaround for PowerScale KB article
+		// https://www.dell.com/support/kbdoc/en-us/000019414/quotas-on-synciq-source-directories
```

Comment:
I like this. Linking to KB article or similar as a comment helps with transparency and understanding of code blocks. 👍